apps/core/src/dashboards/dashboards.controller.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  NotFoundException,
+  Param,
+  Post,
+  Put,
+} from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+
+import { DashboardsService } from './dashboards.service';
+import { CreateDashboardDto } from './dto/create-dashboard.dto';
+import { UpdateDashboardDto } from './dto/update-dashboard.dto';
+import { DeleteDashboardParams } from './params/delete-dashboard.params';
+import { ReadDashboardParams } from './params/read-dashboard.params';
+import { UpdateDashboardParams } from './params/update-dashboard.params';
+
+@ApiTags('dashboards')
+@Controller('dashboards')
+export class DashboardsController {
```

Comment:
what's the benefits of combining the method controllers into a single controller?